### PR TITLE
make age not mandatory, change create patient permission

### PR DESCRIPTION
**Pull request by anroopak**
This PR handles 2 things
1. In patient create/update age and dob is not mandatory, only one need to be there.
2.  adds extra permission to patient create,update and delete. Only `verified` users will be allowed going forward.

### CODE REVIEW

**Comment by vigneshhari** [comment]
i already made a change in the models to allow null values. I didn't see this PR, sorry about that :(

File: care/facility/api/serializers/patient.py [hunk 1]
```diff
@@ -74,6 +74,12 @@ def validate_facility(self, value):
             raise serializers.ValidationError("facility not found")
         return value
 
+    def validate(self, attrs):
+        validated = super().validate(attrs)
+        if not self.partial and not validated.get("age") and not validated.get("date_of_birth"):
+            raise serializers.ValidationError({"date_of_birth": [f"Either age or date_of_birth should be passed"]})
```
Comment (afzalIbnSH) [inline care/facility/api/serializers/patient.py:80]: Making the key "non_field_errors" might be more standardised.

File: care/users/migrations/0022_auto_verify_users_with_facility.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.11 on 2020-04-12 05:00
+
+from django.db import migrations
+
+
+def auto_verify_users_with_facility(apps, *args):
```
Comment (vigneshhari) [inline care/users/migrations/0022_auto_verify_users_with_facility.py:6]: Codacy Issue found: [Unused argument 'args'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5343091)

File: care/users/migrations/0022_auto_verify_users_with_facility.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    # all users who has at-least one facility associated with them is verified
+    for user in user_model.objects.all():
+        user.verified = user.facility_set.exists()
+        user.save()
+
+    # all users who are equal or above District Lab Admin (25)
+    user_model.objects.filter(user_type__gte=25).update(verified=True)
+
+
+def reverse_auto_verify_users_with_facility(*args):
```
Comment (vigneshhari) [inline care/users/migrations/0022_auto_verify_users_with_facility.py:18]: Codacy Issue found: [Unused argument 'args'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5343091)

**Pull request merged**